css/index.css

```diff
@@ -1 +1,573 @@
+  justify-content: space-between;
+  margin-right: 10%;
+}
+
+header nav a {
+  font-weight: bold;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  margin-left: 0;
+}
+
+header nav a:last-child {
+  margin-right: 0;
+}
+
+header img.logo {
+  margin-top: 25px;
+}
+
+/* cta */
+.cta {
+  width: 100%;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.cta .cta-text {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  text-align: center;
+  font-size: 7.7rem;
+  width: 50%;
+}
+
+.cta .cta-text button {
+  background: white;
+  margin-top: 6px;
+  width: 17rem;
+  height: 3.2rem;
+  font-size: 1.7rem;
+  border: 1px solid black;
+  cursor: pointer;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  align-self: flex-start;
+}
+
+.main-content {
+  border-top: 2px solid black;
+  margin-top: 75px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  /* padding: 0 10px; */
+  justify-content: space-between;
+}
+
+.top-content .text-content {
+  width: 50%;
+}
+.top-content .text-content:first-child {
+  margin-right: 2rem;
+}
+.bottom-content .text-content:first-child {
+  margin-right: 2rem;
+}
+.bottom-content .text-content:last-child {
+  margin-left: 2rem;
+}
+.bottom-content .text-content {
+  width: 33.33%;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  margin-top: 45px;
+  width: 100%;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  /* padding: 0 1%; */
+  display: flex;
+  justify-content: space-between;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 2px solid black;
+  padding: 39px 0 0 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 25px 0 2rem 0;
+  text-align: center;
+}
+
+/* services */
+.services-header-img {
+  display: flex;
+  margin: 22px 0 30px 0;
+  width: 100%;
+}
+
+article.services p {
+  line-height: 1.3;
+}
+
+.service-descriptions {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  border-top: 2px solid black;
+  padding-top: 40px;
+  margin-top: 40px;
+}
+
+.service-box {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 42rem;
+  height: 25rem;
+  border: 1px solid black;
+  background: #f4f3f4;
+  padding: 2rem;
+  margin-bottom: 40px;
+}
+
+.service-box h3 {
+  margin: 0;
+}
+
+article.services h2 {
+  font-size: 2.6rem;
+  font-weight: bold;
+}
+
+.service-box a {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid black;
+  background-color: white;
+  height: 3.4rem;
+  width: 14rem;
+  border-radius: 5px;
+  text-decoration: none;
+  font-size: 1.2rem;
+  color: black;
+}
+
+/* service bottom content */
+.services .bottom-content {
+  display: flex;
+  flex-direction: row;
```

Comment:
What's the direction for (since the default is already `row`)?

Reply:
I'd also try to get the image within that `section` on point (currently, the size is looking a bit small and shrunk width-wise).